src/proposal.c

```diff
@@ -2226,6 +2322,26 @@ int Move_ExtFossilSPRClock (Param *param, int chain, RandLong *seed, MrBFlt *lnP
             brlens[v->index] = v->length * (rate[v->index] + rate[v->anc->index]) / 2.0;
             brlens[u->index] = u->length * (rate[u->index] + rate[u->anc->index]) / 2.0;
         }   /* end tk02 branch rate parameter */
+        else if ( subParm->paramType == P_WNBRANCHRATES)
+        {
+            /* adjust prior ratio */
+            if (subParm->paramType == P_WNBRANCHRATES)
+                var = *GetParamVals (modelSettings[subParm->relParts[0]].wnvar, chain, state[chain]);
+            else
+                var = *GetParamVals (modelSettings[subParm->relParts[0]].mixedvar, chain, state[chain]);
```

Comment:
These four lines should probably be replaced by just one line, line 2329. This is the only line that will be executed anyway.

Reply:
I would just leave it as is. Maybe it will be useful when more relaxed clock models are included in the reversible jump.